colly.go

```diff
@@ -269,6 +269,13 @@ func (c *Collector) OnHTML(goquerySelector string, f HTMLCallback) {
 	c.lock.Unlock()
 }
 
+// OnHTML deregister a function. Function will not be execute after register
```

Comment:
Please adjust the function name in comment, it is `OnHTMLDetach` and not `OnHTML`